.travis.yml

```diff
@@ -1,22 +1,32 @@
 language: node_js
 dist: trusty
-sudo: false
+sudo: required
 node_js:
   - "6"
 addons:
   apt:
     packages:
     - lua5.1
     - libxml2-utils
+    - python3
+
 env:
-  - CARTO=0.18.0 MAPNIK='3.0.0 3.0.12'
+  - CARTO=0.18.0 MAPNIK='3.0.0 3.0.12' TRAVIS_PYTHON_VERSION=3.6
+
+before_install:
+  - pyenv global system 3.6 # Workaround for travis-ci/issues/8363
+
 install:
+  - sudo apt-get install -y python3-yaml
   - npm install carto@$CARTO
   - mkdir -p data/world_boundaries data/simplified-land-polygons-complete-3857 data/ne_110m_admin_0_boundary_lines_land data/land-polygons-split-3857
   - touch data/simplified-land-polygons-complete-3857/simplified_land_polygons.shp data/ne_110m_admin_0_boundary_lines_land/ne_110m_admin_0_boundary_lines_land.shp data/land-polygons-split-3857/land_polygons.shp
 script:
   # We're using pipes in the checks, so fail if any part fails
   - set -o pipefail
+  # Check the Taginfo-project
+  - cd ./scripts/taginfo-project && ./generate-taginfo-project-file.py  && cd ../..
+  - git diff taginfo-openstreetmap-carto.json
```

Comment:
It's better to generate to a different file and diff the existing taginfo-openstreetmap-carto.json against the new one, or have the script write to stdout and use `<()`